controller/users.go

```diff
@@ -157,6 +193,137 @@ func (c *UsersController) getKeycloakProfileInformation(ctx context.Context, tok
 	return response, err
 }
 
+func (c *UsersController) updateUserInDB(id *uuid.UUID, ctx *app.UpdateUserAsServiceAccountUsersContext) error {
```

Comment:
What about creating a new Identity/user? I mean the case when some user is being logged in the very first time in Auth and there is no such user in the WIT yet. We should create a new user/identity in WIT if it doesn't exist.

Reply:
I was adding another endpoint to create user/identity  ( and then have AUTH call WIT to check first if user exists before updating ). 
but I could have just 1 endpoint.. Is that what you also imply?

Reply:
It's fine if it's two endpoints: POST /users/:id to create and PATCH /users/:id to update. It's preferable actually. But we have to cover both cases before we start switching UI to Auth.

Reply:
doing that in this PR .. https://github.com/fabric8-services/fabric8-wit/pull/1635 ( yet to push any real stuff.. so review it later please ;)